src/js/clickable-component.js

```diff
@@ -158,6 +164,13 @@ class ClickableComponent extends Component {
   disable() {
     this.addClass('vjs-disabled');
     this.el_.setAttribute('aria-disabled', 'true');
+    if (typeof this.tabIndex_ !== 'undefined') {
+      this.el_.removeAttribute('tabindex');
```

Comment:
We had to enforce consistent use of `tabIndex` rather than `tabindex` throughout the code - it doesn't make any difference here, but it would be better for consistency.